client/src/RoutesTree.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Routes, Route, Redirect } from "react-router-dom";
```

Comment:
Did you intend to import Switch instead?   "Routes" does not seem to be a named export from 'react-router-dom'